etna/packages/etna-js/utils/cancellable_helpers.jsx

```diff
@@ -3,14 +3,10 @@ import {Cancellable, cancelledAsMaybe} from "./cancellable";
 import {useState, useEffect, useCallback} from 'react';
 
 export function useAsyncCallback(fn, deps, cleanup = () => null) {
-  const [_, setContext] = useState(() => new Cancellable());
```

Comment:
Previously, I was keeping the state of the "current cancellable context" in a React state hook.  There was a problem I'll show below.